src/common/cuda_utils.h

```diff
@@ -175,6 +185,18 @@ inline const char* CurandGetErrorString(curandStatus_t status) {
         << "cuRAND: " << common::cuda::CurandGetErrorString(e); \
   }
 
+#define CUDA_1D_KERNEL_LOOP(i, n)                            \
```

Comment:
Function 'CUDA_KERNEL_LOOP' was defined in 'mxnet_op.h'. It's same to the 'CUDA_1D_KERNEL_LOOP'.